aspnetcore/fundamentals/minimal-apis/samples/WebMinAPIs/Program.cs

```diff
@@ -420,32 +420,30 @@ public static string Hello()
 
 app.Run();
 #endregion
-#elif CON
-#region snippet_con
+#elif PBG
+#region snippet_pbg
 var builder = WebApplication.CreateBuilder(args);
 
 // Added as service
 builder.Services.AddSingleton<Service>();
 
 var app = builder.Build();
 
-app.MapGet("/{id}", (int id, int page, Service service) => { });
+app.MapGet("/{id}", (int id,
+                     int page,
+                     [FromHeader(Name = "X-CUSTOM-HEADER")] string customHeader,
+                     Service service) => { });
 
 class Service { }
 #endregion
 
-#elif OV
-#region snippet_ov
+#elif PBP
+#region snippet_pbp
 var builder = WebApplication.CreateBuilder(args);
 
-// Added as service
-builder.Services.AddSingleton<Service>();
```

Comment:
Why this change?

Reply:
The example right above this one for GET now shows resolving a service from DI, so I removed it here to shorten the example and save the repetition.